src/fsharp/vs/ServiceLexing.fs

```diff
@@ -485,7 +483,9 @@ type FSharpLineTokenizer(lexbuf: UnicodeLexing.Lexbuf,
     let skip = false   // don't skip whitespace in the lexer 
     
     let mutable singleLineTokenState = SingleLineTokenState.BeforeHash
-    let fsx = CompileOps.IsScript(filename)
+    let fsx = match filename with
+              | null -> false
```

Comment:
Please change filename to be a string option instead of using null.  Generally avoid using null strings in F# except at interop boundaries